### PR TITLE
build: add pull request template

### PR DESCRIPTION
**Pull request by kacperzolkiewski**
The template looks like this:

# Summary

Explain the **motivation** for making this change: here are some points to help you:

- What issues does the pull request solve? Please tag them so that they will get automatically closed once the PR is merged
- What is the feature? (if applicable)
- How did you implement the solution?
- What areas of the library does it impact?

## Test Plan

Provide **clear steps so another contributor can reproduce the behavior or verify the feature works**.  
For example:

- Steps to reproduce the bug (if this is a bug fix)
- Steps to verify the new feature
- Expected vs. actual results
- Any special conditions or edge cases to test

## Screenshots / Videos

Include any visual proof that helps reviewers understand the change — UI updates, bug reproduction or the result of the fix.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- What areas of the library does it impact?
+
+## Test Plan
+
+Provide **clear steps so another contributor can reproduce the behavior or verify the feature works**.  
+For example:
+
+- Steps to reproduce the bug (if this is a bug fix)
+- Steps to verify the new feature
+- Expected vs. actual results
```
Comment (exploIF) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- Expected vs actual results
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please follow the template so that the reviewers can easily understand what the code changes affect -->
+
+# Summary
+
+Explain the **motivation** for making this change: here are some points to help you:
+
+- What issues does the pull request solve? Please tag them so that they will get automatically closed once the PR is merged
```
Comment (exploIF) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- What issues does the pull request solve? Please tag them so that they will get automatically closed once PR is merged
```
